### PR TITLE
AMI Updates for EKS 1.15.11 

### PR DESCRIPTION
**Pull request by shabir61**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Chili-Man — CHANGES_REQUESTED** [review]
Please update the kernel to 4.19 https://aws.amazon.com/about-aws/whats-new/2019/07/amazon-linux-2-extras-provides-aws-optimized-versions-of-new-linux-kernels/

File: eks-worker-al2.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "binary_bucket_region": "us-west-2",
     "kubernetes_version": null,
     "kubernetes_build_date": null,
-    "docker_version": "19.03.6ce-1.amzn2",
+    "docker_version": "18.09.9ce-2.amzn2",
```
Comment (Chili-Man) [inline eks-worker-al2.json]: why are we downgrading docker


 it should  be this 
```suggestion
    "docker_version": "19.03.6ce-2.amzn2",
```

File: eks-worker-al2.json [hunk 2]
```diff
@@ -14,9 +14,9 @@
...
     "cni_version": "v0.6.0",
-    "cni_plugin_version": "v0.7.6",
+    "cni_plugin_version": "v0.7.5",
```
Comment (Chili-Man) [inline eks-worker-al2.json]: why are you downgrading this as well

**Review by Chili-Man — CHANGES_REQUESTED** [review]

**Review by sakomws — DISMISSED** [review]
lgtm

**Review by Chili-Man — APPROVED** [review]

**Pull request merged**